plugins/resizimg/trumbowyg.resizimg.js

```diff
@@ -14,6 +14,7 @@
     var ResizeWithCanvas = function () {
         // variable to create canvas and save img in resize mode
         this.resizeCanvas = document.createElement('canvas');
+        document.head.appendChild(this.resizeCanvas);
```

Comment:
Why in head?
This should be removed from DOM at some point I think

Reply:
add / remove canvas on setup / reset functions call